app/models/grda_warehouse/tasks/client_cleanup.rb

```diff
@@ -464,22 +464,48 @@ def choose_best_pronouns dest_attr, source_clients
       dest_attr
     end
 
+    # Get the best SSN (has value and quality is full or partial, oldest breaks the tie)
+    # note, source clients might be just an array of hashes
+    SSN_DQS = [1, 2, 8, 9, 99].to_set.freeze
```

Comment:
Can we lean on our friend `HudHelper`?  `HudHelper.util.ssn_data_quality_options.keys`